host/xtest/xtest_main.c

```diff
@@ -104,10 +104,20 @@ ADBG_SUITE_ENTRY(XTEST_TEE_20523, NULL)
 
 ADBG_SUITE_DEFINE_END()
 
+
+ADBG_SUITE_DECLARE(XTEST_TEE_BENCHMARK)
+
+ADBG_SUITE_DEFINE_BEGIN(XTEST_TEE_BENCHMARK, NULL)
+ADBG_SUITE_ENTRY(XTEST_TEE_BENCHMARK_1001, NULL)
+ADBG_SUITE_ENTRY(XTEST_TEE_BENCHMARK_1002, NULL)
+ADBG_SUITE_ENTRY(XTEST_TEE_BENCHMARK_1003, NULL)
+ADBG_SUITE_DEFINE_END()
+
 static const char gdevname_tz[] = "opteearmtz00";
 char *_device = (char *)gdevname_tz;
 unsigned int level = 0;
 static const char glevel[] = "0";
+static const char gsuitename[] = "sanity";
```

Comment:
The test suite is a bit more than a sanity test suite, "regression" is more accurate.